python/trezorlib/tests/device_tests/test_msg_eos_signtx.py

```diff
@@ -0,0 +1,1132 @@
+# This file is part of the Trezor project.
+#
+# Copyright (C) 2012-2018 SatoshiLabs and contributors
+#
+# This library is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License version 3
+# as published by the Free Software Foundation.
+#
+# This library is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the License along with this library.
+# If not, see <https://www.gnu.org/licenses/lgpl-3.0.html>.
+
+import pytest
+import time
+import json
+
+from collections import namedtuple
+from trezorlib import eos, messages as proto
```

Comment:
also please don't use `messages as proto` here either. but you probably won't need to use `messages` at all if you use the input flow methods

Reply:
Done

Reply:
Imported EosSignedTx alone, just to assert instance